### PR TITLE
Session resizing

### PR DESCRIPTION
**Pull request by VoxSecundus**
This PR aims to replicate the session renaming functionality, but with the appropriate behaviour for resizing sessions.

### CODE REVIEW

File: app/models.rb [hunk 1]
```diff
@@ -259,6 +259,14 @@ def rename(name:)
     end
   end
 
+  def resize(size:)
+    if DesktopCLI.resize_session(id, size: size, user: user, remote_host: remote_host).success?
```
Comment (benarmston) [inline app/models.rb:263]: This doesn't match with the method signature: `def resize_session(id, geometry:, remote_host:)`.

Looks like `resize` should be `geometry` and the method signature is missing a `user` param.

File: app.rb [hunk 2]
```diff
@@ -67,6 +67,10 @@
 helpers do
   attr_accessor :current_user
 
+  def size_param
+    params[:size]
```
Comment (benarmston) [inline app.rb:71]: This isn't the right parameter name: https://github.com/openflighthpc/flight-desktop-webapp/pull/90/files#diff-d45b1674ab37689f6ca6dc634fe29959c463fc30cb46746c3b187de88432ae4fR116

**Review by benarmston — CHANGES_REQUESTED** [review]
I can't get this working @VoxSecundus.  Specifying a geometry when the session is started is ignored.  Attempting to resize a running session results in a `500` error.

I've added comments below where I think the issues lie.

**Comment by VoxSecundus** [comment]
I originally used both `size` and `geometry` when implementing the buttons, before refactoring to make it a ubiquitous `geometry`. These must have slipped through the cracks.

**Review by benarmston — CHANGES_REQUESTED** [review]
Another issue @VoxSecundus

File: app/models.rb [hunk 3]
```diff
@@ -259,6 +259,14 @@ def rename(name:)
     end
   end
 
+  def resize(geometry:)
+    if DesktopCLI.resize_session(id, geometry: geometry, user: user, remote_host: remote_host).success?
+      true
+    else
+      raise InternalServerError.new(details: 'failed to resize the session')
```
[the remark this answers is not in the document]
Comment (VoxSecundus) [reply app/models.rb:266]: Shall I also change the other `InternalServerError` calls that use `details` erroneously?

**Comment by benarmston** [comment]
LGTM.

**Pull request merged**